src/main/java/se/sundsvall/incidentmapper/service/IncidentService.java

```diff
@@ -188,7 +193,7 @@ private void updatePobUser(final IncidentEntity entity) {
 	}
 
 	private void updatePobComment(final IncidentEntity entity, final Issue jiraIssue) {
-		StreamSupport.stream(jiraIssue.getFields().getComments().spliterator(), false)
+		jiraIssue.getFields().getComments().stream()
 			.filter(comment -> comment.getCreated().isAfter(entity.getLastSynchronizedPob()))
```

Comment:
I think that this will throw an exception if lastSynchronizedPob is null (at least according to OffsetDateTIme.isAfter javadoc)